edx_proctoring/admin.py

```diff
@@ -104,13 +108,121 @@ def queryset(self, request, queryset):
             return queryset
 
 
+class ProctoredExamListFilter(admin.SimpleListFilter):
+    """
+    Quick filter to allow admins to see which reviews have not been reviewed internally
+    """
+
+    title = _('active proctored exams')
+
+    parameter_name = 'exam__id'
+
+    def lookups(self, request, model_admin):
+        """
+        List of values to allow admin to select
+        """
+
+        now_utc = datetime.now(pytz.UTC)
+        thirty_days_ago = now_utc - timedelta(days=30)
+        one_week_from_now = now_utc + timedelta(days=7)
+
+        # only consider proctored (not practice) exams that have a due date of no more than
+        # a month ago as well as a week into the future. This is to help keep the list of
+        # quick filters small and reasonable in length
+        exams = ProctoredExam.objects.filter(
+            is_proctored=True,
+            is_active=True,
+            is_practice_exam=False,
+            due_date__gte=thirty_days_ago,
+            due_date__lte=one_week_from_now
+        )
+
+        lookups = (())
+
+        for exam in exams:
+            lookups += ((exam.id, exam.exam_name),)
```

Comment:
Would it be better showing just the exam name? We could end up having three "Midterm"s in the list, not knowing which course it belongs to.